### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by dxvladislavvolkov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,6 +44,22 @@ Add DevExtreme to an Angular application (follow [this](https://github.com/DevEx
 > devextreme add devextreme-angular
 ```
 
+### <a name="add-devextreme-react"></a>React 
+
+Add DevExtreme to an React application:
```
Comment (TikhonovaTatyana) [inline README.md]:
```suggestion
Add DevExtreme to a React application:
```

File: README.md [hunk 2]
```diff
@@ -42,6 +44,22 @@ Add DevExtreme to an Angular application (follow [this](https://github.com/DevEx
...
+
+#### <a name="add-devextreme-react-no-global"></a>Without global package usage
```
Comment (TikhonovaTatyana) [inline README.md]: I suggest 
"Package installed locally" and "Package installed globally" 
and change it in all other places

File: README.md [hunk 3]
```diff
@@ -80,6 +98,32 @@ Add a view to an Angular application with the DevExtreme layout template:
...
+### <a name="devextreme-app-react"></a>React
+
+#### <a name="devextreme-app-react-no-global"></a>Without global package usage
+
+Create a new DevExtreme application:
+```bash
+> npx -p devextreme-cli devextreme new react-app app-name [--layout][--empty]
+```
+
+Add a view to an React application with the DevExtreme layout template:
```
Comment (TikhonovaTatyana) [inline README.md]:
```suggestion
Add a view to the DevExtreme application:
```

File: README.md [hunk 4]
```diff
@@ -80,6 +98,32 @@ Add a view to an Angular application with the DevExtreme layout template:
...
+```
+
+#### <a name="devextreme-app-react-global"></a>Global package usage
+
+Create a new DevExtreme application:
+```bash
+> devextreme new react-app app-name [--layout][--empty]
+```
+
+Add a view to an React application with the DevExtreme layout template:
```
Comment (TikhonovaTatyana) [inline README.md]:
```suggestion
Add a view to the DevExtreme application:
```

File: README.md (hunk 2 above)
Comment (dxvladislavvolkov) [reply README.md]: Maybe "Package installed globally" and "Without installing the package globally"
@nightskylark What do you think?

Comment (nightskylark) [reply README.md]: It's not about working with locally installed packages. It's about runtime packages installation with `npx`.

Comment (nightskylark) [reply README.md]: @dxvladislavvolkov It' s OK

Comment (TikhonovaTatyana) [reply README.md]: OK, then, let's it leave as is.

**Pull request merged**